File: app/src/main/java/com/jds/fitnessjunkiess/getfitapp/data/pojo/WorkoutExercise.java
package com.jds.fitnessjunkiess.getfitapp.data.pojo;

import android.arch.persistence.room.ColumnInfo;

public class WorkoutExercise {
  @ColumnInfo(name = "id")
  public  int id;
  @ColumnInfo(name = "name")
  public  String exerciseName;
  @ColumnInfo(name = "picture")
  public  String picture;
  @ColumnInfo(name = "instructions")
  public  String instructions;
  @ColumnInfo(name = "exerciseType")
  public  String exerciseType;
  @ColumnInfo(name = "exerciseMuscleGroups")
  public  String exerciseMuscleGroups;
  @ColumnInfo(name = "exerciseId")
  public  int exerciseId;
  @ColumnInfo(name = "workoutId")
  public  int workoutId;
  @ColumnInfo(name = "userId")
  public  int userId;
  @ColumnInfo(name = "lengthTime")
  public  String lengthTime;
  @ColumnInfo(name = "restTime")
  public  String restTime;
  @ColumnInfo(name = "sprintTime")
  public  String sprintTime;
  @ColumnInfo(name = "sets")
  public  int sets;
  @ColumnInfo(name = "reps")
  public  int reps;
}

File: app/src/main/java/com/jds/fitnessjunkiess/getfitapp/data/pojo/WorkoutExercise.kt
package com.jds.fitnessjunkiess.getfitapp.data.pojo

import android.arch.persistence.room.ColumnInfo

data class WorkoutExercise(
    @ColumnInfo(name = "id")
    var id: Int = 0,
    @ColumnInfo(name = "name")
    var exerciseName: String = "",
    @ColumnInfo(name = "picture")
    var picture: String = "",
    @ColumnInfo(name = "instructions")
    var instructions: String = "",
    @ColumnInfo(name = "exerciseType")
    var exerciseType: String = "",
    @ColumnInfo(name = "exerciseMuscleGroups")
    var exerciseMuscleGroups: String = "",
    @ColumnInfo(name = "exerciseId")
    var exerciseId: Int = 0,
    @ColumnInfo(name = "workoutId")
    var workoutId: Int = 0,
    @ColumnInfo(name = "userId")
    var userId: Int = 0,
    @ColumnInfo(name = "lengthTime")
    var lengthTime: String = "",
    @ColumnInfo(name = "restTime")
    var restTime: String = "",
    @ColumnInfo(name = "sprintTime")
    var sprintTime: String = "",
    @ColumnInfo(name = "sets")
    var sets: Int = 0,
    @ColumnInfo(name = "reps")
    var reps: Int = 0
)
